PendingReleaseNotes

```diff
@@ -228,6 +228,10 @@
   to ensure compatibility during upgrades, but can be disabled once old usage logs
   are no longer present to avoid performance overhead.
 
+* RBD: The "start-time" argument for RBD snapshot and purge schedules has been
+  removed because it produced confusing results and conflicted with upcoming
+  work to stagger scheduled operations to avoid activity spikes.
```

Comment:
The release note needs to be moved somewhere close to the top of the file, so that it comes before `>=20.0.0` section -- this is happening in umbrella, not in tentacle.